include/flexflow/config.h

```diff
@@ -23,7 +23,7 @@
 #include <cudnn.h>
 #elif defined(FF_USE_HIP_ROCM)
 #include <hipblas.h>
-#include <miopen.h>
+#include <miopen/miopen.h>
```

Comment:
@eddy16112 @jiazhihao Sorry, last question. Is changing this header path acceptable? When I install the latest rocm version, this is the path to the miopen header. Maybe `<miopen.h>` is the previous location of the header and is this change ok?

Reply:
If the directory of MIOPEN is changed, I am OK with your changes.

Reply:
Ok great! Thank you :)